docs/index.md

```diff
@@ -2362,14 +2362,14 @@ or the [source](https://github.com/mochajs/mocha/blob/master/lib/mocha.js).
 [discord-mocha]: https://discord.gg/KeDn2uXhER
 [emacs]: https://www.gnu.org/software/emacs/
 [emacs-mocha.el]: https://github.com/scottaj/mocha.el
-[example-babel]: https://github.com/mochajs/mocha-examples/tree/master/packages/babel
+[example-babel]: https://github.com/mochajs/mocha-examples/tree/main/packages/babel
 [example-connect-test]: https://github.com/senchalabs/connect/tree/master/test
```

Comment:
Looks like you missed changing this  `main`  for this one *line_2366* & _line_2367_

Reply:
Thanks for the spot! That's an external repo that still uses `master`, not us. 🙂